web/src/pages/your_account/send_transaction_dialog.rs

```diff
@@ -0,0 +1,158 @@
+use std::{borrow::Cow, mem};
+
+use at2_ns::ThinUser;
+use material_yew::{
+    dialog::{ActionType, MatDialogAction},
+    MatButton, MatDialog, MatFormfield, MatList, MatListItem, WeakComponentLink,
+};
+use yew::{prelude::*, worker::Agent};
+
+use super::transaction_builder::TransactionBuilder;
+use crate::agents;
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Properties {
```

Comment:
Please add some short comments for each of the public structures. Like `golint` which is expecting to have a comment for all exported elements.